src/npc_class.cpp

```diff
@@ -267,7 +267,7 @@ distribution distribution::dice_roll( int sides, int size )
     } );
 }
 
-distribution distribution::operator+( const distribution &other ) const
+distribution distribution::sum( const distribution &other ) const
```

Comment:
I agree, `operator+/+=` types are pretty safe.
It only gets confusing when the operator does something ambiguous, like variable access.

Though it would be pretty consistent if `sum` always meant returning a new object (it "returns a sum") and `add` would modify the current one (since it can't "return an add").